app/models/host.rb

```diff
@@ -90,7 +90,10 @@ class Host < ApplicationRecord
                             :inverse_of => :host
   has_many                  :host_aggregate_hosts, :dependent => :destroy
   has_many                  :host_aggregates, :through => :host_aggregate_hosts
-
+  
+  # Physical infra reference
+  has_one :physical_server, :foreign_key => "serial_number", :primary_key => "service_tag", :class_name => "PhysicalServer"
```

Comment:
This is a repeat of the commits in https://github.com/ManageIQ/manageiq/pull/14026